test/unit/ngsi-ld/examples/contextRequests/updateContextStaticLinkedAttributes.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "@context": "http://context.json-ld",
+    "luminosity": {
+      "type": "Property",
+      "value": 87,
+      "providedBy": {
```

Comment:
are you sure of this "self relationship"?

Reply:
The test defines:

```javascript
  deepClonedObject.types.Lamp.staticAttributes[0].link = {
                attributes: ['luminosity'],
                name: 'providedBy',
                type: 'Building'
            };
```

`staticAttributes[0]` is the `controlledAsset`, so it now has a `link` subproperty.

Therefore when the `luminosity` is set to `87`:

```
{
    "@context": "http://context.json-ld",
    "id": "urn:ngsi-ld:Building:001",
    "type": "Building",
    "luminosity": {
      "type": "Property",
      "value": 87,
      "providedBy": {
        "type": "Relationship",
        "object": "urn:ngsi-ld:Lamp:lamp1"
      }
    }
```

Is upserting the linked entity - i.e. the `urn:ngsi-ld:Building:001` altering the  `luminosity`  attribute and maintaining a `providedBy` relationship of a property.

Reply:
LGTM